src/test/java/rest/gorestcoIn/utils/helpermethods/APICal.java

```diff
@@ -0,0 +1,35 @@
+package rest.gorestcoIn.utils.helpermethods;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.http.Header;
+import io.restassured.response.Response;
+import io.restassured.response.ValidatableResponse;
+
+public class APICal {
+
+    public static Response post(String jsonBody, Header header) {
+
+        return RestAssured
+                .given()
+                .header(header)
+                .contentType(ContentType.JSON)
+                .body(jsonBody)
+                .post("users")
```

Comment:
it's obvious, your method designed specially to post user, usually methods should have wider usage, so the endpoint should be method parameter, instead of hard coded one